fixtures/output/Output_builder.py

```diff
@@ -0,0 +1,67 @@
+class Output_builder():
```

Comment:
Please use pep8 conventions to name your classes, underscores are not allowed in class names, this link can be helpful ---> https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Thank you very much, Shalu, I'll work on it and revert back to you as soon as possible.